.reuse/dep5

```diff
@@ -1,3 +1,15 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: disasm-util
 Source: https://github.com/sorru94/disasm-util
+
+Files: Cargo.lock
```

Comment:
In my opinion .license files are better: they are more "local" since they are kept close to the licensed files. In general dep5 is quite a workaround.